interface/patient_file/front_payment.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php global $srcdir, $pid, $encounter, $cryptoGen;
```

Comment:
Above line seems odd. Why need to do this?

Reply:
prob prevent IDE from complaining otherwise not good practice!

Reply:
https://github.com/openemr/openemr/pull/6899#:~:text=%3C%3Fphp%20global%20%24srcdir%2C%20%24pid%2C%20%24encounter%2C%20%24cryptoGen%3B
PHPStorm suggested this.

Reply:
Prob best to remove that redline feature in PHPStorm or to ignore it :)